### DISCUSSION TITLE
Navbar Example: Nested Menu

### DISCUSSION THREAD

**Discussion by mazipan** [mantinedev/mantine#1539] [Feature requests]
In the current documentation, we have NavBar on the AppLayout/AppShell section which has an example of how to add NavBar to your App.

See: https://mantine.dev/core/app-shell/#navbar-and-aside-components

<img width="821" alt="Screen Shot 2022-05-30 at 09 08 06" src="https://user-images.githubusercontent.com/7221389/170904472-8334ba35-1afb-4c83-9951-8da826f8c943.png">

In the real common study case, we usually will have a nested menu inside the NavBar.
It's good if we have an official example in how to create this kind of Nested Navbar menu using Mantine.

**Comment by rtivital** [comment]
https://ui.mantine.dev/component/navbar-nested
